content/en/docs/setup/best-practices/certificates.md

```diff
@@ -147,10 +187,10 @@ The value of `<nodeName>` for `kubelet.conf` **must** match precisely the value
 1. Run `kubectl` as follows for each config:
 
 ```shell
-KUBECONFIG=<filename> kubectl config set-cluster default-cluster --server=https://<host ip>:6443 --certificate-authority <path-to-kubernetes-ca> --embed-certs
-KUBECONFIG=<filename> kubectl config set-credentials <credential-name> --client-key <path-to-key>.pem --client-certificate <path-to-cert>.pem --embed-certs
-KUBECONFIG=<filename> kubectl config set-context default-system --cluster default-cluster --user <credential-name>
-KUBECONFIG=<filename> kubectl config use-context default-system
+   KUBECONFIG=<filename> kubectl config set-cluster default-cluster --server=https://<host ip>:6443 --certificate-authority <path-to-kubernetes-ca> --embed-certs
+   KUBECONFIG=<filename> kubectl config set-credentials <credential-name> --client-key <path-to-key>.pem --client-certificate <path-to-cert>.pem --embed-certs
+   KUBECONFIG=<filename> kubectl config set-context default-system --cluster default-cluster --user <credential-name>
+   KUBECONFIG=<filename> kubectl config use-context default-system
```

Comment:
Why the extra indentation here?

Reply:
It's a poor attempt to have that content indent under the numbered list item, I guess. I think it should indent under the list, but that was not the correct way to do it. I can fix that.

Reply:
I went ahead and removed the spacing. It doesn't really need to indent, I guess.